tracer/src/Datadog.Trace.Tools.Runner/RunCiCommand.cs

```diff
@@ -339,8 +339,12 @@ private async Task ExecuteAsync(InvocationContext context)
                         if (CoverageUtils.TryCombineAndGetTotalCoverage(codeCoveragePath, outputPath, out var globalCoverage, useStdOut: false) &&
                             globalCoverage is not null)
                         {
-                            // Adds the global code coverage percentage to the session
-                            session.SetTag(CodeCoverageTags.PercentageOfTotalLines, globalCoverage.GetTotalPercentage());
+                            // We only report the code coverage percentage if the customer manually sets the 'DD_CIVISIBILITY_CODE_COVERAGE_ENABLED' environment variable according to the new spec.
+                            if (EnvironmentHelpers.GetEnvironmentVariable(Configuration.ConfigurationKeys.CIVisibility.CodeCoverage).ToBoolean() == true)
```

Comment:
Do we explicitly want to check the env var like this, or should we be checking `CIVisibilitySettings.CodeCoverageEnabled` 🤔 I know the spec says env var, but is that definitely what we want given we can have other sources too?